src/main/java/de/teamlapen/vampirism/core/tags/ModDamageTypeTags.java

```diff
@@ -7,7 +7,6 @@
 import org.jetbrains.annotations.NotNull;
 
 public class ModDamageTypeTags {
-
```

Comment:
As general note for Pull Request, please refrain from doing cleanup on otherwise not referenced code. This just makes reviewing harder. Splitting this in a separate PR is totally fine.